src/common/vscode/ide/vscodeUtil.ts

```diff
@@ -0,0 +1,24 @@
+import * as vscode from "vscode";
+
+const VERSION_SEPARATOR = ".";
+
+function isVscodeVersionSufficient(requiredVersion: string): boolean {
+    const splittedRequiredVersion = requiredVersion.split(VERSION_SEPARATOR).map(str => Number.parseInt(str, 10));
+    const splittedCurrentVersion = vscode.version.split(VERSION_SEPARATOR).map(str => Number.parseInt(str, 10));
+
+    for (let digitIndex = 0; digitIndex < splittedRequiredVersion.length; digitIndex++) {
+        if (splittedCurrentVersion.length <= digitIndex) {
+            return true;
+        } else if (splittedCurrentVersion[digitIndex] < splittedRequiredVersion[digitIndex]) {
+            return false;
+        } else if (splittedCurrentVersion[digitIndex] > splittedRequiredVersion[digitIndex]) {
+            return true;
+        }
+    }
+
+    return splittedCurrentVersion.length >= splittedRequiredVersion.length;
+}
```

Comment:
I didn't want to add a new dependency for a few line taking task. Is it ok to leave itlike this?

Reply:
sure! Since this is just a simple task, as you said no dependency needed. If you need further development on versioning you might want to look for it though.